package.json

```diff
@@ -132,6 +132,7 @@
     "@storybook/addon-interactions": "^7.6.16",
     "@storybook/addon-links": "^7.6.16",
     "@storybook/addon-styling": "^1.3.7",
+    "@storybook/jest": "^0.2.3",
```

Comment:
Does `@storybook/test` work? `@storybook/jest` is being deprecated in favor of /test

Reply:
Not sure if it'll work or not with our setup. But probably worth checking

Reply:
Sure, I can try that. The linting recommended installing this 🤔